tests/test_read_edf.py

```diff
@@ -0,0 +1,15 @@
+from pathlib import Path
+
+from electos.ballotmaker import read_edf
```

Comment:
In the source files you are using this instead. I think the form which imports the function directly is more readable, but I don't think it matters which form you choose as long as it's consistent, especially since the module and function names are the same.

```suggestion
from electos.ballotmaker.read_edf import read_edf
```

Reply:
Agreed and updated.